pharmaverse/aNCA
# Refactor: Simplified the way shiny app is run from within the package

## Issue

Closes #83 

## Description

Removed `app_server.R` and `app_ui.R` files, the initialization and import are contained in single file `run_app.R`. The function loads `system.file()` with folder containing shiny files and calls `shiny::runApp()` from that directory.

All `@importFrom` statements related to shiny were moved to this single file and sorted alphabetically for better management. 

## How to test

Please install the package and call `run_app()` to confirm that the app starts up without issues.

## Contributor checklist
